docs/getting_started/basics/authoring.md

```diff
@@ -1,36 +1,44 @@
 
-## Creating Your First Node
+## Creating a Node
 
-To create a new [operator](../intro/terminology.md/#operator), you can either press the `Tab Key` or `Double Left-Click` with your mouse.  
+To create a new [node](../terminology.md/#operator), press the `Tab` key or `Double Left-Click` to open the "*Create Node*" dialog. This dialog enables searching for nodes by name and/or filtering by [catalog](../terminology.md/#catalog). The nodes will be listed alphabetically underneath, where they can be added to the Shift board by double-clicking on them.
 
-![Create Node Dialog](../images/node_creation_dialog.png "Create Node Dialog")  
+![Create Node Dialog](../../images/create_node_dialog.gif "Create Node Dialog")   
 
-Then you can choose the type of operator you want to create and double-click on its name to create it.
+Alternatively, nodes can be added to the board using the [Nodes List widget](../ui_overview.md/#operator) present in the Shift interface.
 
 ## Creating Connections
 
-If you want a certain operator to pass information to another one, you can connect their [plugs](../intro/terminology.md/#plug). To do that, you just need to `Left-Click and Drag` from one plug to the other plug you want to connect to. This will propagate whatever data is produced by the first operator and pass it to the connected plug.  
+To establish a connection between plugs from different nodes, `Left-Click` and `Drag` from one plug to the desired plug. By creating a connection, the workflow can propagate the data produced in the output plugs of a node and transfer it to the connected input plug of another node.
 
-![Connected Nodes](../images/connected_nodes.png "Connected Nodes")  
-
-This will have created the simplest example of a Shift workflow.
+![Connected Nodes](../../images/connect_plugs.gif "Connected Nodes")  
 
 ## Creating New Plugs
 
-Certain nodes in Shift allow the creation of new plugs. To do that you can `Right-Click` on the node and select the "*Create New Plug*" option if available. It will prompt you to a new dialog asking to fill in the information needed to create the plug.
+Certain nodes in Shift allow the creation of new plugs. To create a new plug, `Right-Click` on the node to open its context menu and select the "*Create New Plug*" option if available. This will open the "*Add Plug*" dialog, which contains all the information fields required to create a custom plug. These are:
+
+- **Name**: The name of the plug. This must be unique among the plugs of the same direction in the same node. For example, if an input plug named *value* already exists in the node, no other input plug can have that name. Allowed plug names follow the same restrictions as variable names in Python.
+- **Type**: The type of the plug. This will determine which data type values are compatible with the plug. More details about plug types are provided [here](#TODO).
```

Comment:
Remember once https://github.com/Inbibo/shift_docs/pull/29 it's merged, replace this `TODO` for pointing to the correct page:
docs/reference/nodes.md (This contains the plug type specification in the `Plugs Type` section)

Reply:
Done